airflow2/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM apache/airflow:2.0.1-python3.8
+
+USER airflow
+RUN pip install --no-cache-dir --user tika elasticsearch
```

Comment:
we can add a few more dependencies, we know will be used in the future. 

`mlflow nympy pandas pytorch  scikit-learn spacy `

for a start